app/upgrades/v19/upgrades.go

```diff
@@ -93,13 +93,6 @@ func CreateUpgradeHandler(
 			}
 		}
 
-		ctxCache, writeFn = ctx.CacheContext()
-		if err := UpdateBaseFee(ctxCache, fee); err == nil {
```

Comment:
fee market keeper is now unused so it can be removed from the upgrade handler

Reply:
Should also update the `app.go`